hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestDataUtil.java

```diff
@@ -83,10 +92,7 @@ public static OzoneBucket createVolumeAndBucket(OzoneClient client,
 
     objectStore.createVolume(volumeName, volumeArgs);
 
```

Comment:
Nit: Can we remove these empty lines in this and above functions?

Reply:
Removed.